.gitignore

```diff
@@ -0,0 +1,25 @@
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+
+# OS generated files #
+######################
+.DS_Store
```

Comment:
#### `REQUIRED`

Consider adding files not related to the project to `.gitignore`

> Files not related to the project should not be commited and added to .gitignore.

:bulb: Add `bin/.DS_Store` and ` lib/.DS_Store`, alternatively you can use  `*.DS_Store` instead, this will target all files with `.DS_Store` and add to `.gitignore`.